packages/shared/lib/contexts/governance/utils/createProposalFromError.ts

```diff
@@ -0,0 +1,28 @@
+import { get } from 'svelte/store'
+
+import { IAccount } from '@core/account/interfaces'
+import { networkStatus } from '@core/network/stores'
+
+import { ProposalErrorMode, ProposalStatus } from '../enums'
+import { IProposal } from '../interfaces'
+import { getProposalStatusForMilestone } from './getProposalStatusForMilestone'
+
+export function createProposalFromError(
+    proposal: IProposal,
```

Comment:
change this to `IProposalMetadata` because the status is calculated in this function

Reply:
Also `account` doesn't seem to be used.